src/inverseRobot.js

```diff
@@ -8,6 +8,19 @@
 
 function inverseRobot(robot) {
   // write code here
+  const correctRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (correctRobot[value] !== undefined) {
```

Comment:
Replace with `correctRobot[value]`, it will be the same as `correctRobot[value] !== undefined`